src/examples/using-with-cmake/example.cpp

```diff
@@ -10,12 +10,20 @@
 ///
 //-----------------------------------------------------------------------------
 
-#include <iostream>
-
 #include "axom/core/utilities/About.hpp"
+#include "fmt/fmt.hpp"
+
+#include <iostream>
 
 int main(int argc, char **argv)
 {
+   // Using fmt library exported by axom
+   std::cout << fmt::format(
+        "Example of using and installed version of axom v{}.{}.{}-{}",
+        AXOM_VERSION_MAJOR, AXOM_VERSION_MINOR,
+        AXOM_VERSION_PATCH, AXOM_VERSION_EXTRA) << std::endl << std::endl;
```

Comment:
You shouldn't have to use `fmt` for this. 

Given that the intent of this is to provide a simple example of how to link `Axom` to a downstream `CMake` application, I prefer we remove anything extraneous and keep things simple.

This could be simplified to:
```
std::cout << AXOM_VERSION_FULL << "-" << AXOM_VERSION_PATH << std::endl;
```

Reply:
Thanks @gzagaris -- I intentionally added `fmt` to this example to ensure that it is being properly exported and as an example of how to use this.  (See PR description).

Side note: Once we address #209 the ``fmt`` usage will have to be guarded.

Reply:
OK -- maybe we should add advertisement banners in our examples too :)

That's fine, but technically it would be better if it was a separate test, e.g., `test-fmt-export.cpp` or something like that. I would imagine we'd want to have similar tests for all our other built-in TPLs as well, not just `fmt`.